Inspectors/Inspect-MSTeamsLinkPreview.json

```diff
@@ -0,0 +1,17 @@
+{
+	"FindingName": "Microsoft Teams Users Allowed to Preview Links in Messages",
+	"Description": "Microsoft Teams by default enables and allows users to preview links in messages. Some organizations may wish to disable this functionality. Changing these settings may have unintended consequences. Speak with shareholders and understand what functionality may be affected before disabling this access.",
```

Comment:
should change "shareholders" to "stakeholders.

Reply:
Done